assembly/src/parsers/io_ops.rs

```diff
@@ -335,11 +339,11 @@ fn parse_hex_value(op: &Token, param_str: &str, param_idx: usize) -> Result<u64,
 /// Determines the minimal type appropriate for provided value and returns appropriate instruction
 /// for this value
 fn build_push_one_instruction(value: u64) -> Result<Node, ParsingError> {
-    if value <= u8::MAX as u64 {
+    if u8::try_from(value).is_ok() {
```

Comment:
This could be further simplified with a match, I think.  But clippy ain't smart enough.  (And I don't want to go through everything and fix it manually.  This is supposed to be a clippy-only PR.)